src/components/HeaderMenu/index.ts

```diff
@@ -0,0 +1,3 @@
+import HeaderMenu from './HeaderMenu';
```

Comment:
We don't need files like this. They are unuseful.
We can rename HeaderMenu.tsx -> index.tsx
And the same for Spinner.tsx.